# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/websocket/moz.build
# Commit: 74bb3c2ab767
# Full Hash: 74bb3c2ab767a67e3e67f9781ce13ea2cebca918
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-07-16 21:27:37
# Regressor Bug: 1497249
# File Overlap Count: 4
# Description:
#   Bug 1497249 - P1: Introduce nsWebSocketConnection interface for separating socket manipulation r=michal,necko-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30023
# ==============================================================================

diff -r 68618a10778e -r 74bb3c2ab767 netwerk/protocol/websocket/moz.build
--- a/netwerk/protocol/websocket/moz.build	Thu Jul 16 13:31:27 2020 +0000
+++ b/netwerk/protocol/websocket/moz.build	Thu Jul 16 12:59:23 2020 +0000
@@ -10,6 +10,7 @@
 XPIDL_SOURCES += [
     'nsITransportProvider.idl',
     'nsIWebSocketChannel.idl',
+    'nsIWebSocketConnection.idl',
     'nsIWebSocketEventService.idl',
     'nsIWebSocketListener.idl',
 ]
@@ -31,6 +32,7 @@
 UNIFIED_SOURCES += [
     'BaseWebSocketChannel.cpp',
     'IPCTransportProvider.cpp',
+    'nsWebSocketConnection.cpp',
     'WebSocketChannel.cpp',
     'WebSocketChannelChild.cpp',
     'WebSocketChannelParent.cpp',